.eslintrc.js

```diff
@@ -30,16 +37,17 @@ module.exports = {
     'no-console': 0,
     'no-bitwise': 0,
     'max-classes-per-file': 0,
+    'max-len': ['error', { code: 130 }],
```

Comment:
The other changes in this file are not real configuration changes (only removing unnecessary quotes to satisfy the linter).  This is the one rule configuration change.  Since there are multiple instances of lines longer than 100 chars, I thought it made sense to relax this rule to allow this.